### PR TITLE
refactor drawBody and drawBodyTrails to support face, body, tail style

### PR DESCRIPTION
**Pull request by psugihara**
part of #59 

png face style next

### CODE REVIEW

**Comment by psugihara** [comment]
Screenshot 2024-03-22 at 10 20 09 PM
Screenshot 2024-03-22 at 10 19 54 PM

File: src/visuals.js [hunk 1]
```diff
@@ -25,17 +24,17 @@ export const Visuals = {
     if (!this.showIt) return
 
     // when there are 3 or more bodies, step the simulation
-    if (this.bodies.filter(b => !b.life || b.life > 0).length >= 3) {
+    if (this.bodies.filter((b) => !b.life || b.life > 0).length >= 3) {
```
Comment (psugihara) [inline src/visuals.js:27]: not sure why prettier has fixes in here again

Comment (okwme) [reply src/visuals.js:27]: i also tried turning off all of my format-on-save settings and just let prettier take over but haven't noticed consistency

Comment (psugihara) [reply src/visuals.js:27]: I recommend installing the prettier plugin to format on save.

Comment (psugihara) [reply src/visuals.js:27]: I think this was from merging in some pre-prettier stuff. 🤔🧐

**Pull request merged**